tests/test_other.py

```diff
@@ -10256,3 +10256,9 @@ def test_output_name_collision(self):
     self.assertExists('hello.wasm')
     self.assertExists('hello_.wasm')
     self.assertContained('hello, world!', self.run_js('hello.wasm'))
+
+  def test_EM_PYTHON_MULTIPROCESSING(self):
```

Comment:
If this really is just a temporary thing perhaps mention in changelog why it was added and that is just a transitional thing to help people temporarily revert?